app/src/main/res/values/environment_config.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="app_scheme">keyapp</string>
```

Comment:
it's not environment config

Reply:
where to move it
can you suggest?

Reply:
@Davran331122 ^^ ?

Reply:
maybe create one more not translatable xml file. deeplink.xml